src/leader_confirmation_service.rs

```diff
@@ -28,26 +28,23 @@ pub struct LeaderConfirmationService {
 impl LeaderConfirmationService {
     fn get_last_supermajority_timestamp(
         bank: &Arc<Bank>,
-        leader_id: Pubkey,
```

Comment:
is there no test for this "exclude the leader from our calculation" case that's broken by this change?

Reply:
Oh there is a test apparently. Something failed. I'll get to it later tonight. Afk now :)